### ISSUE TITLE
Issue with 

### ISSUE THREAD

**Issue by mulqueenr** [icbi-lab/nextNEOpi#32]
Hi,
Thanks for your tool! 
I'm having trouble running a small test batch where I get the error:

[main] DEBUG nextflow.Session - Session aborted -- Cause: No such property: merged_fastq_ch for class: Script_39ee2431

when supplied paired end fastq files. The supplied fastq files seem to be properly formatted and properly paired.
Attached is the log file. Any help would be appreciated! Thank you!
Ryan

[nextflow.log](https://github.com/icbi-lab/nextNEOpi/files/12033672/nextflow.log)

**Comment by riederd** [comment]
Thanks for reporting. I've not seen this before. I realize that you are using a newer nextflow version compared to ours. I'll need to see if it is related to this. Can you meanwhile also send me your `batchFile`?

**Comment by riederd** [comment]
The newest version of nextflow does not support DSL1, which was used to write `nextNEOpi`, can you try to run the pipeline as follows:

```
NXF_VER=22.10.8 nextflow run nextNEOpi.nf <your options here>
```

**Comment by mulqueenr** [comment]
Hi, Thanks for the quick reply! We're getting a new error now that might have to do with version support. I'm trying a simple case with unchanged params file. But I attached them both for clarity. Thanks again for the support!

`CAPSULE: Downloading dependency io.nextflow:nf-commons:jar:22.10.8
CAPSULE: Downloading dependency org.codehaus.groovy:groovy:jar:3.0.13
CAPSULE: Downloading dependency ch.grengine:grengine:jar:1.3.0
CAPSULE: Downloading dependency io.nextflow:nf-httpfs:jar:22.10.8
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-nio:jar:3.0.13
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-templates:jar:3.0.13
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-xml:jar:3.0.13
CAPSULE: Downloading dependency io.nextflow:nextflow:jar:22.10.8
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-json:jar:3.0.13
N E X T F L O W ~ version 22.10.8
Unable to parse config file: '/rsrch4/home/genetics/athennavan/resources/params.config'
 Compile failed for sources FixedSetSources[name='/groovy/script/Script3B8B0DA90C115D0CC3C7C6D800112797/_nf_config_39c3dca3']. Cause: BUG! exception in phase 'semantic analysis' in source unit '/groovy/script/Script3B8B0DA90C115D0CC3C7C6D800112797/_nf_config_39c3dca3' Unsupported class file major version 64`

[NewBatchfile_TEST.csv](https://github.com/icbi-lab/nextNEOpi/files/12052530/NewBatchfile_TEST.csv)
[params.config.txt](https://github.com/icbi-lab/nextNEOpi/files/12052531/params.config.txt)

**Comment by riederd** [comment]
closed by b9ec9002846c6d3dd8e186730515d55e8e7d00b5

Feel free to open a new issue in case v1.4 is still failing
